### DISCUSSION TITLE
Matching parentheses / brackets not highlighted

### DISCUSSION THREAD

**Discussion by tommyalatalo** [LazyVim/LazyVim#223] [General]
I'm wondering if this is something that I have coming from my config or if it's not there by default - if I hover my cursor over an open parenthesis or bracket I don't get the closing one highlighted. This is a really small detail but it actually affects me when I'm working as I realized that I often look at these when jumping with `%`. 

Is any of the included plugins with lazyvim possibly altering the highlight display, and how does one troubleshoot this?

**Comment by folke** [comment]
If you were one of the early lazyvim users, then I accidentally disabled matchit and matchparen in Lua/config/lazy.lua

Just comment those lines there in that file for disabled plugins.

**Reply by tommyalatalo** [reply]
> If you were one of the early lazyvim users, then I accidentally disabled matchit and matchparen in Lua/config/lazy.lua
> 
> Just comment those lines there in that file for disabled plugins.

That was it :) I take pride in being a day 1 victim :D
